integration/docker/entrypoint.sh

```diff
@@ -47,6 +50,8 @@ for keyvaluepair in $(env | grep "ALLUXIO_"); do
   if [[ ! "${special_env_vars[*]}" =~ "${key}" ]]; then
     confkey=$(echo ${key} | sed "s/_/./g" | tr '[:upper:]' '[:lower:]')
     echo "${confkey}=${value}" >> conf/alluxio-site.properties
+  else
+    echo "${key}=${value}" >> conf/alluxio-env.sh
```

Comment:
Some keys require `export` (such as AWS credentials), do you think we should fix that and stick with key=value?